aptos-move/framework/aptos-framework/sources/account/account.move

```diff
@@ -698,17 +746,32 @@ module aptos_framework::account {
 
     /// Revoke any signer capability offer in the specified account.
     public entry fun revoke_any_signer_capability(account: &signer) acquires Account {
+        let offerer_addr = signer::address_of(account);
+        if (features::is_default_account_resource_enabled()) {
+            assert!(
+                resource_exists_at(offerer_addr),
+                error::not_found(ENO_SUCH_SIGNER_CAPABILITY),
+            );
+        } else {
+            assert!(exists_at(offerer_addr), error::not_found(EACCOUNT_DOES_NOT_EXIST));
+        };
         check_offering_permission(account);
```

Comment:
in above function this is before the check, here it is below, let's make it consistent.